lib/interface/cli/commands/hybrid/init.cmd.js

```diff
@@ -214,11 +220,11 @@ const initCmd = new Command({
             }
 
             if (_.isUndefined(shouldMakeDefaultRe)) {
-                if (_.get(sdk, 'config.context.isNoAuth') || await isNewAccount()) {
+                if (!_.get(sdk, 'config.context.isNoAuth') && await isNewAccount()) {
```

Comment:
fixed bug where the runtime would always be set to default if there is not .cfconfig file, even if the account is not new